### PR TITLE
Fix marking LTI module items viewed

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13840
affects: student
release note: Fixed marking LTI module items as viewed

Test plan:
* narmstrong.instructure.com : student2
* Dashboard > Modules > Modules > Sequential
* The items should require to be viewed but are not viewed
* Tap the items then tap Launch External Tool
* Module items should be marked as viewed when you go back to the items list

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019478&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed marking LTI module items as viewed

#### Affected Apps: Student

[MBL-13840](https://instructure.atlassian.net/browse/MBL-13840)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.58% | 93.56% | 0.03%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CanvasCore/CanvasCore/Modules/ModuleItem/ModuleItem+Progress.swift [hunk 1]
```diff
@@ -61,7 +61,7 @@ extension ModuleItem {
             case let .externalURL(url):
                 progress = Progress(kind: kind, contextID: contextID, itemType: .url, itemID: url.absoluteString)
             case let .file(id):
-                progress = Progress(kind: kind, contextID: contextID, itemType: .externalTool, itemID: id)
+                progress = Progress(kind: kind, contextID: contextID, itemType: .file, itemID: id)
```
Comment (shichongrui) [inline CanvasCore/CanvasCore/Modules/ModuleItem/ModuleItem+Progress.swift]: Good catch.

**Pull request merged**